hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/AsyncSocket.java

```diff
@@ -105,7 +106,7 @@ public final long readEvents() {
      *
      * @return the remote address.
      */
-    public final SocketAddress remoteAddress() {
+    public final SocketAddress getRemoteAddress() {
```

Comment:
Can you also change getter setter names for the above functions? ioBuffersWritten -> getIoBuffersWritten() etc.

Reply:
sure